data/assets.json

```diff
@@ -4662,5 +4665,116 @@
+      }
+    },
+    "coingecko": "",
+    "coinmarketcap": ""
+  },
+  {
+    "symbol": "stATOM-INIT LP",
+    "name": "",
+    "precision": 6,
+    "logo": [
+      "https://next.list.initia.tech/images/0x92af68b117c6aff7dcac7547c9c72cf7e0c2b9044fa20148b7839a289394be51.webp",
+      "https://next.list.initia.tech/images/0x8e4733bdabcf7d4afc3d14f0dd46c9bf52fb0fce9e4b996c939e195b8bc891d9.webp"
+    ],
+    "description": "",
+    "slugs": [],
+    "type": "native",
+    "id": {
+      "initia": {
+        "stone-11": "move/b9d795966ffd06125f71307f73b8ca22c5af5026fae64d6110af610735e999df"
+      }
+    },
+    "coingecko": "",
+    "coinmarketcap": ""
+  },
+  {
+    "symbol": "MaticX-INIT LP",
+    "name": "",
+    "precision": 6,
+    "logo": [
+      "https://next.list.initia.tech/images/0x76f75544f1867066b0c64cfa71d91463767fb117bfc01cadf7e2fb5cfa204ba7.webp",
+      "https://next.list.initia.tech/images/0x8e4733bdabcf7d4afc3d14f0dd46c9bf52fb0fce9e4b996c939e195b8bc891d9.webp"
+    ],
+    "description": "",
+    "slugs": [],
+    "type": "native",
+    "id": {
+      "initia": {
+        "stone-11": "move/c9c9cb27844944ed08a8441a2dec30a46366c6f8c0a9b8fed028fcdd8a6ea273"
+      }
+    },
+    "coingecko": "",
+    "coinmarketcap": ""
+  },
+  {
+    "symbol": "NOVA-USDC LP",
+    "name": "",
+    "precision": 6,
+    "logo": [
+      "https://next.list.initia.tech/images/0x29824d952e035490fae7567deea5f15b504a68fa73610063c160ab1fa87dd609.webp",
+      "https://next.list.initia.tech/images/0xf5129d0d9af3f7c30a9f522495b7164d52ca6ccaa5d317b67c7ad328b1244a29.webp"
+    ],
+    "description": "",
+    "slugs": [],
+    "type": "native",
+    "id": {
+      "initia": {
+        "stone-11": "move/ce1d9a305fd808ac81fa0bc9dd63c3b78775d41c20e2d65f2323397ad9ac159c"
+      }
+    },
+    "coingecko": "",
+    "coinmarketcap": ""
+  },
+  {
+    "symbol": "USDC-INIT LP",
+    "name": "",
+    "precision": 6,
+    "logo": [
+      "https://next.list.initia.tech/images/0x29824d952e035490fae7567deea5f15b504a68fa73610063c160ab1fa87dd609.webp",
+      "https://next.list.initia.tech/images/0x8e4733bdabcf7d4afc3d14f0dd46c9bf52fb0fce9e4b996c939e195b8bc891d9.webp"
+    ],
+    "description": "",
+    "slugs": [],
+    "type": "native",
+    "id": {
+      "initia": {
+        "stone-11": "move/dbf06c48af3984ec6d9ae8a9aa7dbb0bb1e784aa9b8c4a5681af660cf8558d7d"
+      }
+    },
+    "coingecko": "",
+    "coinmarketcap": ""
+  },
+  {
+    "coingecko": "wrapped-steth",
+    "coinmarketcap": "",
+    "description": "",
+    "id": {
+      "initia": {
+        "stone-11": "move/e3a97600010bb368be99d6f4ed231aca4dbba9b6340edf3c48dfbfd0474e92e8"
+      }
+    },
+    "logo": "https://next.list.initia.tech/images/0xe3a97600010bb368be99d6f4ed231aca4dbba9b6340edf3c48dfbfd0474e92e8.webp",
+    "name": "Wrapped Staked Ether",
+    "precision": 6,
+    "slugs": ["lido", "ethereum"],
+    "symbol": "wstETH",
+    "type": "native"
   }
```

Comment:
there's already an existing entry for this for previous testnets. should merge it into that one